docs/user/openstack/privileges.md

```diff
@@ -0,0 +1,51 @@
+# Required Privileges
+
+In order to install an OpenShift cluster on OpenStack successfully, the user provided to the installer needs certian privileges. The easiest way to achieve this level of permission and ensure success is to install with a user who has administrative privileges.
```

Comment:
certain

Reply:
I don't think that a _Keystone user_ has to get `admin` _Keystone role_ into its Keystone project.
Example given, I'm currently a regular user of Red Hat's internal OpenStack Cloud (PSI), and I only have the `member` role into my _Keystone project_ and I can deploy OpenShift just fine. I would create the necessary resources (VMs, volumes, networks, FIPs, etc) without permission error.

If I understand your wording here is that it would be required to obtain `admin` role in the project but I don't think that would be 1) necessary and 2) secure for the cloud admins to give admin power to their users.